lib/iris/tests/unit/fileformats/netcdf/loader/test__get_cf_var_data.py

```diff
@@ -25,16 +24,24 @@ def setUp(self):
         self.shape = (300000, 240, 200)
         self.expected_chunks = _optimum_chunksize(self.shape, self.shape)
 
-    def _make(self, chunksizes):
-        cf_data = mock.Mock(_FillValue=None)
+    def _make(self, chunksizes=None, shape=None, dtype="i4"):
```

Comment:
Perhaps specify dtype= np.dtype("i4")

Reply:
Actually, I now see that this is OK.
we have a line below which "regularises" this, i.e. [line 35 `dtype = np.dtype(dtype)`](https://github.com/SciTools/iris/blob/3dabecae5f10874fc7b460a2b4e290f0155e0a36/lib/iris/tests/unit/fileformats/netcdf/loader/test__get_cf_var_data.py#L35)
So that makes it OK :  `_make(...).dtype` is always a proper np.dtype object, and the default is `np.dtype('i4')`.